src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -50,20 +50,6 @@ public function doSomething(\Foo\Bar $foo)
     {
     }
 }
-'
-                ),
-                new CodeSample(
-                    '<?php
-
-use Foo\Bar;
-use Foo\Bar\Baz;
-
-class SomeClass
-{
-    public function doSomething(\Foo\Bar $foo): \Foo\Bar\Baz
```

Comment:
I would rather keep code sample with return type, than the one without it 😉.

Reply:
👍